docs/backend/guides/_category_.json

```diff
@@ -0,0 +1,5 @@
+{
+  "label": "Guides",
+  "position": 3,
+  "collapsed": true
+}
```

Comment:
This is a placeholder for when this gets merged in so we can begin with other sections. Backend Guides cannot easily be worked on until this PR is merged.